src/animationLoop.js

```diff
@@ -74,16 +68,31 @@ const prototype = {
     }
 
     while (animationLoop.accumulatedTime > 0) {
-      animationLoop.state.forEach(animationLoop.step);
+      state.forEach(animationLoop.step);
       animationLoop.accumulatedTime -= timeStep;
     }
 
-    // Render and filter in one iteration.
-    filterRight(
-      animationLoop.state,
-      renderSubscriber,
-      1 + animationLoop.accumulatedTime / timeStep
-    );
+    const alpha = 1 + animationLoop.accumulatedTime / timeStep;
+    let index = 0;
+    let nextState;
+    let subscriber;
+
+    while (index < state.length) {
+      subscriber = state[index];
```

Comment:
If you inline the `let` here babel will transform it into a regular `var`. Probably _slightly_ more readable.